Dockerfile

```diff
@@ -3,6 +3,8 @@ RUN mkdir -p /go/src/github.com/asecurityteam/ipam-facade
 WORKDIR $GOPATH/src/github.com/asecurityteam/ipam-facade
 COPY --chown=sdcli:sdcli . .
 RUN sdcli go dep
+RUN go get -u github.com/gobuffalo/packr/v2/packr2
+RUN packr2
```

Comment:
suggestion: I believe this would be a better fit for the `build` task rather than the Dockerfile.

Reply:
I agree, but we don't have control over how our Docker images get built in Dockerhub, so similar to installing the dependencies on line 5, this is a workaround that limitation.